### ISSUE TITLE
Change Crossing/Crosswalk Presets

### ISSUE THREAD

**Issue by homersimpsons** [openstreetmap/iD#3244]
I think that those both presets are 2 confusing, and need to be merge in only once.
[This issue](https://github.com/osmandapp/Osmand/issues/2791) can convince you. I know that I have often tag only zebra.

Then I think the best solution is to disable this while [wiki](http://wiki.openstreetmap.org/wiki/Key:crossing) is unclear, because there could be zebra+traffic_signals, zebra+island....

**Comment by bhousel** [comment]
This comes up a lot, but we're not going to change it anytime soon.
see #2918, #3049 

tl;dr:  
* We picked values for the presets that matched the most common `crossing` tags in use.  That's why the crosswalk tag uses `crossing=zebra`. 
* The osm wiki for the crossing tag does not document the tags the way they are used in practice.  [see taginfo for real usage](https://taginfo.openstreetmap.org/keys/crossing#values)
* Changing the wiki does not update the data.  If the community wants to improve this tag, they need to make a real plan to update the data in OSM.
* `controlled/uncontrolled` is a legal concept that depends on where you live, not on the visible ground truth.  (In many places crosswalk markings are considered "control devices")
